packages/mobile/src/page/Place/Detail/index.tsx

```diff
@@ -1,44 +1,24 @@
 import { LeftArrow } from "@components/atoms/icon";
 import ChipGroup from "@components/molecules/ChipGroup";
 import FullPageModalTemplate from "@components/templates/FullPageModalTemplate";
-import { PlaceSchoolDto } from "@shared/swagger-api/generated";
-import { useSchoolQuery } from "src/hooks/api/school";
+import ErrorFallback from "src/components/atoms/ErrorFallback";
+import SuspenseFallback from "src/components/atoms/SuspenseFallback";
+import AsyncBoundary from "src/components/templates/AsyncBoundary";
 import useSearch from "src/hooks/useSearch";
-import DetailGroup from "src/page/Place/DetailGroup";
+import PlaceDetailBody from "src/page/Place/Detail/PlaceDetailBody";
 import { useAppDispatch } from "src/store";
 import { setHashMenu } from "src/store/placeSlice";
 
 import { menuList } from "../../../__mocks__/index";
-import $ from "./style.module.scss";
 
 function PlaceDetail() {
   const dispatch = useAppDispatch();
-  const position = useSearch({ target: "position" })!;
-  const currentPosition =
-    position === "all" ? undefined : position.split("")[0].toUpperCase();
-  const {
-    data: schoolData,
-    isLoading: schoolLoading,
-    isError: schoolError,
-  } = useSchoolQuery({
-    area: currentPosition as PlaceSchoolDto["school"]["area"],
-  });
-  if (schoolLoading) return <div>로딩중입니다.</div>;
-  if (schoolError) return <div>에러가 발생했습니다.</div>;
-  if (schoolData === undefined)
-    return <div>캠퍼스 장소 리스트 불러오기 실패</div>;
-
-  const handleMenu = () => {
-    dispatch(setHashMenu({ hashString: position }));
-  };
-
-  const schoolDatas = schoolData.filter((item: PlaceSchoolDto) => {
-    return (
-      item?.school.area === currentPosition || currentPosition === undefined
-    );
-  });
 
-  const checkMenu = (position: string) => {
+  const bodyHeight =
+  "calc(var(--vh, 1vh) * 100 - (env(safe-area-inset-top) + env(safe-area-inset-bottom)))";
+  const position = useSearch({ target: "position" })!;
+  
+    const checkMenu = (position: string) => {
     switch (position) {
```

Comment:
```suggestion
    switch (position) {
      case "north":
        return 1;
      case "east":
        return 2;
      case "south":
        return 3;
      default:
        return 0;
    }
```

이부분은 지금 수정하신 건 아니지만 합칠수 있을것같아요!